src/mpyl/steps/build/docker_build.py

```diff
@@ -133,3 +137,9 @@ def execute(self, step_input: Input) -> Output:
             message=f"Failed to build docker image for {step_input.project.name}",
             produced_artifact=None,
         )
+
+    @staticmethod
+    def substitute_pr_number(original_value: Optional[str], pr_number: Optional[int]):
+        if original_value and pr_number:
+            return original_value.replace("{PR-NUMBER}", str(pr_number))
+        return original_value
```

Comment:
nit, can be done in one line:

```suggestion
        return original_value.replace("{PR-NUMBER}", str(pr_number)) if original_value and pr_number else original_value
```

Reply:
Updated! 👍